### ISSUE TITLE
[BUG] There is no consistent ordering of h1 and h2 labels

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5145]
**Describe the bug**
There is no consistent ordering of h1 and h2 labels (because there are none.) AInspector states the purpose of this as “One of the fundamental features of the web is the provision of easy access to useful information. Consistent ordering of h1 and h2 elements used to identify recurring page sections common across all pages in the website will make it easier for people to find information they are seeking and to navigate between and within pages.”
WCAG 3.2.3 Consistent Navigation (Primary)
WCAG 3.2.4 Consistent Identification

**To Reproduce**

1. Go to app.illinois.edu in Firefox
2. Run AINspector
3. Go to the Navigation violations

**Expected behavior**
There should be  consistent ordering of h1 and h2 labels

**Actual behavior**
There is no consistent ordering of h1 and h2 labels

**Screenshots**

**Desktop (please complete the following information):**
PC:
• OS: Windows 11
• Device: Dell laptop
• Browser: Chrome 135.0.7049.116/Firefox 138.0.1

**Additional context**

**Comment by acovher2** [comment]
This is still the case in 7.3.17 version.

**Comment by dobromirdobrev** [comment]
@acovher2 ,

Currently I do not see any Navigation violations with the latest 7.3.21 version. 

<img width="490" height="1160" alt="Image" src="https://github.com/user-attachments/assets/63e8856e-0696-47e3-a32a-1a249f90188c" />

I presume that this should have been fixed alongside with #5144. 

Could you please verify with the latest available version on dev environment? https://api-dev.rokwire.illinois.edu/illinois/
